### PR TITLE
Updating major versions of dependencies

### PR DESCRIPTION
**Pull request by sglanzer**
This is a #major# change to move up the next versions of eslint and associated projects

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,13 +28,13 @@
...
+    "eslint": "^2.1.0",
     "istanbul": "^0.4.2",
     "mocha": "^2.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^4.1.7",
-    "eslint-config-standard": "^4.4.0",
+    "babel-eslint": "^5.0.0",
+    "eslint-config-standard": "^5.1.0",
     "eslint-plugin-standard": "^1.3.1"
```
Comment (job13er) [inline package.json]: The plugin doesn't need updating?

Comment (sglanzer) [inline package.json]: It's up to date - that's the latest

**Comment by job13er** [comment]
I updated the team Developers to have permissions for this repo, so the pull approve should work if you throw another commit on here.

**Comment by sglanzer** [comment]
Cool, I wasn't sure how far you wanted to expand privileges

**Comment by sglanzer** [comment]
Looks like I'm missing a new peer dependency anyway

File: package.json [hunk 2]
```diff
@@ -24,13 +28,13 @@
...
-    "eslint": "^1.10.3",
+    "eslint": "^2.1.0",
     "istanbul": "^0.4.2",
     "mocha": "^2.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^4.1.7",
-    "eslint-config-standard": "^4.4.0",
+    "babel-eslint": "^5.0.0",
+    "eslint-config-standard": "^5.1.0",
```
Comment (job13er) [inline package.json]: Looks like this may require another dependency now

https://travis-ci.org/ciena-frost/eslint-config-frost-standard/builds/109890304

**Comment by sglanzer** [comment]
Good to go?

**Pull request merged**